src/base/definitions.jl

```diff
@@ -100,5 +101,17 @@ const VOLTAGE_BUSES_NO = "voltage_buses_no"
 const CURRENT_BUSES_NO = "current_buses_no"
 const TOTAL_SHUNTS = "total_shunts"
 const AUX_ARRAYS = "aux_arrays"
+const LOOKUP = "lookup"
 
-const SIMULATION_ACCEPTED_KWARGS = [:initial_guess, :initialize_simulation]
+const SIMULATION_ACCEPTED_KWARGS = [:initial_guess, :initialize_simulation, :system_to_file]
+
+PSY.get_V_ref(value::PSY.AVRFixed) = value.Vf
```

Comment:
We should standardize these functions. These hacks should be probably in helper functions also in PSY.